config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "process.env.DB_USER",
```

Comment:
The value for `username` is set as a string instead of referencing the environment variable. Consider using `process.env.DB_USER` directly without quotes to ensure the environment variable is used.